Resources/Prototypes/Atmospherics/reactions.yml

```
@@ -91,6 +91,79 @@
   effects:
   - !type:N2ODecompositionReaction {}
 
+- type: gasReaction
+  id: bzFormation
+  priority: 2
+  maximumTemperature: 313.149 # Atmospherics.FireMinimumTemperatureToExist - 60
+  minimumRequirements:
+  - 0     # oxygen
```

yamllint (1.35.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)